### PR TITLE
Drop support for Python 2

### PR DESCRIPTION
**Pull request by michael-k**

### CODE REVIEW

File: tests/myapp/tests.py [hunk 1]
```diff
@@ -100,16 +97,7 @@ def flush(self):
...
-                    # unicode literals in the doctests screw up doctest on py3.
-                    # this is pretty icky, but I can't find any other
-                    # workarounds :(
-                    text = re.sub(r"""\bu(["\'])""", r"\1", text)
-                    temp.write(b(text))
-                else:
-                    temp.write(text)
-
+                temp.write(text.encode("latin-1"))
                 temp.flush()
```
Comment (matthiask) [inline tests/myapp/tests.py]: Is the temporary file even necessary at all now?

Comment (michael-k) [reply tests/myapp/tests.py]: No, it's not :)

File: docs/conf.py [hunk 2]
```diff
@@ -1,5 +1,4 @@
 # -*- coding: utf-8 -*-
```
Comment (jdufresne) [inline docs/conf.py]: You can drop these coding cookies as well. Python 3 defaults to decoding source files as utf-8.

**Comment by craigds** [comment]
I haven't reviewed the code changes, but no objections to this in principle. It's time.

**Comment by matthiask** [comment]
Thanks @michael-k!

**Comment by goodtune** [comment]
Already merged but no objection either.

**Pull request merged**